### PR TITLE
make manpager.vim self-contained

### PR DESCRIPTION
**Pull request by Konfekt**
As discussed in https://github.com/vim/vim/issues/2323#issuecomment-346626606

this new implementation does no longer depend on `ftplugin/man.vim` and thus is stabler than the ancient hack.

### CODE REVIEW

File: runtime/doc/filetype.txt [hunk 1]
```diff
@@ -706,4 +673,23 @@ You can change the default by defining the variable g:tex_flavor to the format
...
+ZIMBU							*ft-zimbu-plugin*
+
+The Zimbu filetype plugin defines mappings to move to the start and end of
+functions with [[ and ]].
+
+The mappings can be disabled with: >
+	let g:no_zimbu_maps = 1
+<
+
+
```
Comment (chrisbra) [inline runtime/doc/filetype.txt:694]: I suppose this change is unrelated?

Comment (Konfekt) [reply runtime/doc/filetype.txt:694]: Yes.

**Comment by goweol** [comment]
if manpager.vim does not use man.vim, can we remove whole MANPAGER stuff from man.vim?
Then, could you patch the ftplugin/man.vim in your patch?

**Comment by chrisbra** [comment]
since you are the maintainer, you can simply send the complete file for inclusion to Bram. Then it will be included in one of the next runtime file updates.

**Comment by chrisbra** [comment]
I think this was included as of d23a823669d93fb2a570a039

**Comment by chrisbra** [comment]
Sorry, correct commit should have been 7254067ee970686cc3ff4a1effc
